drivers/scheduler/k8s/k8s.go

```diff
@@ -2133,6 +2133,28 @@ func (k *K8s) WaitForRunning(ctx *scheduler.Context, timeout, retryInterval time
 		}
 	}
 
+	t := func() (interface{}, bool, error) {
```

Comment:
nit: maybe rename to `isAppTerminating` or something similar? this func is short enough to read and find out what it does, but a name would've made it easier to read

Reply:
It's a good suggestion. I was just trying to be consistent with how these functions are named in torpedo repo. But will change.